src/markdown-pages/docs/ecosystem/galasa-cli.md

```diff
@@ -0,0 +1,153 @@
+---
+path: "/docs/ecosystem/ecosystem-cli"
+title: "Using the Galasa CLI"
+---
+
+Submit and monitor Galasa test runs in you Galasa Ecosystem by using a command line interface (Galasa CLI). The Galasa CLI lets you submit and monitor test runs by using the same set of commands, regardless of the technology that you are using to run your pipeline. 
+
+[Download the CLI tool](https://github.com/galasa-dev/cli/releases) and [access the code](https://github.com/galasa-dev/cli) from the _cli_ repository in GitHub. 
```

Comment:
Will need to highlight there are 4 versions of galasactl, for the different operating systems and architectures,  needs to list them

Reply:
Done